### PR TITLE
919 spanish readme

### PR DESCRIPTION
**Pull request by woodsoul507**
# Description

*Spanish readme for Flame*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [ X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ X] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [ X] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
- [ X] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [ X] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [ X] I read and followed the [Flame Style Guide].
- [ X] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [ X] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [ X] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [ X] No, this is *not* a breaking change.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database]. Indicate, which of these issues are resolved or fixed by this PR.*

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md

### CODE REVIEW

File: i18n/README-ES.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<p align="center">
+  <a href="https://flame-engine.org">
+    <img alt="flame" width="200px" src="https://user-images.githubusercontent.com/6718144/101553774-3bc7b000-39ad-11eb-8a6a-de2daa31bd64.png">
+  </a>
+</p>
+
+<p align="center">
+Minimalista motor para el desarrollo de videojuegos con Flutter.
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:8]: Motor minimalista *

File: i18n/README-ES.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+---
+
+[English](/README.md) | [简体中文](/i18n/README-ZH.md) | [Polski](/i18n/README-PL.md) | [Русский](/i18n/README-RU.md) | [Español](/i18n/README-ES.md)
+
+---
+
+## Acerca de 1.0.0
+
+Nuestro objetivo es lanzar la v1 pronto. Periódicamente lanzamos RCs (release candidates) a medida que el código evoluciona,
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:26]: lanzar pronto la v1.

File: i18n/README-ES.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+y estamos bastante complacidos con el punto en el que nos encontramos (aunque aún puede haber cambios).
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:27]: complacidos en el punto que nos encontramos (aunque aún pueden existir cambios)

File: i18n/README-ES.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+Por favor usar esta versión como un avance de la nueva versión de Flame y también para aportar feedback al equipo sobre
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:29]: Favor de usar esta

File: i18n/README-ES.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+la nueva estructura y/o características.
+
+La rama `main` es lo más próximo a una migración hacia la v1. La rama `master-v0.x` es el último lanzamiento v0
+(en el cual aún estamos fusionando algunos parches y arreglos críticos).
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:33]: (en la cual aún estamos

File: i18n/README-ES.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+<a title="Pub" href="https://pub.dartlang.org/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
+en pub. La última versión estable hasta ahora es
+<a title="Pub" href="https://pub.dartlang.org/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout" /></a>.
+Siéntase libre de elegir la versión que mejor se adapte a sus necesidades.
+
+---
+
+## Documentación
+
+Tome en cuenta que la documentación en la rama `main` de este repositorio es más nueva que la de la última versión lanzada.
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:45]: Tome en cuenta que la documentación en la rama `main` de este repositorio es más nueva que la última versión lanzada

File: i18n/README-ES.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+[aquí](https://flame-engine.github.io/flame/). También puede examinar el código fuente de estos ejemplos [aquí](https://github.com/flame-engine/flame/tree/main/examples).
+
+El sitio oficial de Flame, el cual también contiene la documentación se puede encontrar
+[aquí](https://flame-engine.org/).
+
+## Ayuda
+
+Tenemos un canal de ayuda en el servidor de Discord Fireslime, únete [aquí](https://discord.gg/pxrBmy4).
+
+También contamos con un [FAQ](FAQ.md), por favor busque respuesta primero allí para resolver sus preguntas.
```
Comment (victormanuelfrancodev) [inline i18n/README-ES.md:64]: por favori busque su respuesta primero ahí

**Review by victormanuelfrancodev — COMMENTED** [review]
I added some comments , please choose the better option. Thank you!

**Review by victormanuelfrancodev — APPROVED** [review]
Thank you!

**Comment by woodsoul507** [comment]
Thanks the comments. I made another PR.

**Pull request closed without merging**